src/linux/backend-v4l2.cpp

```diff
@@ -1060,9 +1112,13 @@ namespace librealsense
                                                 << ", payload size " << buffer->get_length_frame_only();
                                     }
                                     LOG_WARNING("Incomplete frame received: " << s.str()); // Ev -try1
-                                    librealsense::notification n = { RS2_NOTIFICATION_CATEGORY_FRAME_CORRUPTED, 0, RS2_LOG_SEVERITY_WARN, s.str()};
-
-                                    _error_handler(n);
+                                    bool kpi_violated = _kpi_checker.update_and_check(_profile, buf.timestamp);
+                                    if (kpi_violated)
```

Comment:
Can be merged

Reply:
I'd rather let it as is, so that it can be understood that the error is sent because the kpi has been violated.